crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -259,18 +260,36 @@ static_assert(is_assignable_to(Grandchild, Intersection[Child1, Child2]))
 
 static_assert(not is_assignable_to(Parent, Intersection[Parent, Unrelated]))
 static_assert(not is_assignable_to(int, Intersection[int, Not[Literal[1]]]))
+# The literal `1` is not assignable to `str`, so the intersection of int and str is definitely an int that is not `1`
+static_assert(is_assignable_to(Intersection[int, str], Intersection[int, Not[Literal[1]]]))
 static_assert(not is_assignable_to(int, Not[int]))
 static_assert(not is_assignable_to(int, Not[Literal[1]]))
 
 static_assert(not is_assignable_to(Intersection[Any, Parent], Unrelated))
 static_assert(is_assignable_to(Intersection[Unrelated, Any], Intersection[Unrelated, Any]))
 static_assert(is_assignable_to(Intersection[Unrelated, Any], Intersection[Unrelated, Not[Any]]))
 
-# TODO: The following assertions should not fail (see https://github.com/astral-sh/ruff/issues/14899)
-# error: [static-assert-error]
-static_assert(is_assignable_to(Intersection[Any, int], int))
+# Intersection with `Any` is effectively ignored for the sake of assignment
```

Comment:
This comment isn't right. Intersection with `Any` is not ignored, rather it dominates, in the sense that `Any & T` is assignable to any type, regardless of `T`. (`Any & T` is even assignable to a type `S` that is disjoint from `T`, because `Any` could represent `S`, and `S & T` is `Never`, which is assignable to `S`.)
```suggestion
```